tutor/specs/e2e/preview-courses.e2e.ts

```diff
@@ -16,38 +11,35 @@ test.describe('Preview Courses', () => {
             await page.click('testId=show-detail')
             await page.click('testId=create-preview')
         }
-    })
 
-    test('displays a collapsible side panel that creates a course', async ({ page }) => {
         await visitPage(page, '/courses')
         await page.click('testId=preview-course-card')
         await page.click('testId=preview-panel-create-course')
         await expect(page).toHaveSelector('testId=new-course-wizard')
         await expect(page).toHaveSelector('.select-dates')
+
     })
 
-    test('hides preview panel for non-preview courses', async ({ page }) => {
+    test('hides preview panel if the offering is not available for course creation', async ({ page }) => {
+        await findOrCreateTeacherAccount({ page, userName: 'previewteacher' })
+        await page.waitForSelector('.tutor-navbar')
+
         await visitPage(page, '/courses')
         await page.click('testId=preview-course-card')
-
-        await page.goBack()
         await page.evaluate(() => {
             window._MODELS.offerings.array.forEach((a:any) => a.is_available = false)
         })
-        await page.goForward()
-
-        await expect(page).not.toHaveSelector('testId=side-panel >> testId=preview-panel-create-course', { timeout: 100 })
+        await expect(page).not.toHaveSelector('testId=side-panel >> testId=preview-panel-create-course')
     })
 
-    test('hides preview panel for non-available offerings', async ({ page }) => {
+    test('hides preview panel for real courses and the associated preview course', async ({ page }) => {
+        await findOrCreateTeacherAccount({ page, userName: 'teacher01' })
```

Comment:
no need for `findOrCreateTeacherAccount`, we already have `teacher01`,  `teacher02` and review student 01 & 02.  The setup.ts pre-authorizes them and stores their cookies and then switches to using them when `withUser` is called

Reply:
Ah I forgot about loginAs :D thanks!